### ISSUE TITLE
Broken ANSI Color Import

### ISSUE THREAD

**Issue by guevarez30** [Urigo/SOFA#1064]
Line 7

index.mjs
```javascript
import { red, yellow, green, blue } from 'ansi-colors';
```
Error: 
```
import { red, yellow, green, blue } from 'ansi-colors';
                             ^^^^
SyntaxError: Named export 'blue' not found. The requested module 'ansi-colors' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'ansi-colors';
const { red, yellow, green, blue } = pkg;
```

Replaced with 
```javascript
import pkg from 'ansi-colors';
const { red, yellow, green, blue } = pkg;
```

**Comment by ardatan** [comment]
Would you create a PR for this change?

**Comment by ardatan** [comment]
Fixed in 0.11.2
